e2-studio-star-rx72n-firmware/libs/rx_usb_comm/src/rx_usb_comm.c

```
@@ -1252,18 +1165,74 @@ rx_usb_comm_receive(rx_usb_comm_handle_t* handle, rx_frame_t* frame, const uint3
+        return reset_err;
+      }
+
+      if (handle->on_reset_cb != nullptr) {
+        handle->on_reset_cb(frame, handle->cb_ctx);
+      }
+
+      continue;
+    }
+
+    /* PONG / RESET_ACK: consume silently, loop for next frame */
+    if (frame->header.type == k_frame_type_pong || frame->header.type == k_frame_type_reset_ack) {
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)